client/src/components/TestSuite/TestRunButton/TestRunButton.tsx

```diff
@@ -25,7 +19,7 @@ const TestRunButton: FC<TestRunButtonProps> = ({
   testRunInProgress,
   buttonText,
 }) => {
-  const showRunButton = runnableIsTestSuite(runnable) || (runnable as TestGroup).user_runnable;
+  const showRunButton = (runnable as TestGroup).user_runnable !== false;
```

Comment:
```suggestion
  const showRunButton = (runnable as TestGroup).user_runnable
```

Reply:
This is not the same thing because we need undefined to mean 'yes, it is runnable'.  It is only not runnable if it is explicitely set to false.  **If you accept this proposed change the run button will disapear from the suite-level.**  Granted, this probably could be refactored so that suite-level explicitly sets it to true.

This is weird because this line somehow got reverted somewhere along the way though... I already made this change here: https://github.com/inferno-framework/inferno-core/pull/132/files

Reply:
That fix was merged into the old branch for this ticket. That PR was closed, and this fix was migrated into this new branch. I'll add an explanatory comment in the code.